app/src/main/java/com/project/symptoms/activity/MainActivity.java

```diff
@@ -65,7 +73,7 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onResume() {
         super.onResume();
         // This is needed to be done again because of the context used in DateTimeUtils
-        // changes when another view is also registered as picker
+        // changes when another view is also registered as piker
```

Comment:
why this change?

Reply:
Honestly, don't know why that is changed